src/controller/java/src/chip/onboardingpayload/OnboardingPayload.kt

```diff
@@ -55,6 +102,18 @@ class OnboardingPayload(
 ) {
   var optionalQRCodeInfo: HashMap<Int, OptionalQRCodeInfo>
 
+  enum class VendorId(val value: Int) {
+    COMMON(0x0000),
+    UNSPECIFIED(0x0000),
```

Comment:
unspecified and common mapping to the same thing seems not ideal.

How about naming this MATTER_STANDARD to be inline with manufacturers.xml?

Reply:
Looking more at the code, it looks like 0 being named MATTER_STANDARD in manufacturers.xml seems invalid. so maybe say UNSPECIFIED/INVALID but with a comment explaing.

COMMON + UNSPECIFIED being the same alias is confusing though.

Reply:
Changed to UNSPECIFIED(0x0000) and remove COMMON